src/content/docs/en/install/manual.mdx

```diff
@@ -71,7 +71,7 @@ Astro must be installed locally, not globally. Make sure you are *not* running `
   </Fragment>
   <Fragment slot="pnpm">
   ```shell
-  pnpm install astro 
+  pnpm add astro
```

Comment:
I wouldn't replace install here, since this guide is install a brand new project.

Reply:
Got it! Thanks for pointing it out 💡